sources/framework/VEModel/R/models.R

```diff
@@ -180,7 +180,7 @@ getModelRoots <- function(get.root=0,Param_ls=NULL) {
     } else { # happens if ve.runtime is defined but not an absolute pqth (unlikely)
       test.paths <- normalizePath(file.path(roots,modelRoot))
       modelRoot <- test.paths[dir.exists(test.paths)]
-      if ( length(modelRoot)==0 || ! nzchar(modelRoot[1]) ) {
+      if (any( length(modelRoot[1])==0 | ! nzchar(modelRoot[1])) ) {
```

Comment:
We don't need modelRoot[1] for the first part, checking length() -- while fine, it isn't necessary because the result of length is a single value, and the logical test will already be a single T or F

Reply:
Conversation with Jeremy 8/2: using [1] indexing is actually a good solution. Keep the || here, to guarantee that the result of the logical test is an ampersand, but do implement [1] indexing to use the first value if there are multiple values in a vector.